### DISCUSSION TITLE
please remove onyxia attunement

### DISCUSSION THREAD

**Discussion by zadeoglu** [chromiecraft/chromiecraft#3350] [Suggestions]
![image](https://user-images.githubusercontent.com/84640316/163408172-116b3c6b-a9d3-47d4-adf8-411e6fb7757b.png)

**Comment by BeefSteaker** [comment]
Maybe just make it so warlocks can summon inside even without attunement, like with Molten core & BWL.

**Comment by Santaria94** [comment]
Recently, I leveled the mage to 60. Sometimes I am asked to go to Onyxia before or after the MC, since they need ranged fighters there. I generally don't mind helping, but wasting a few hours doing an attune for that? Sorry, but what is the motivation for this? I don't need a T2 helmet because the UBRS turban is better than that. Cloak? Again, craft is better. There is not even good loot for me here. The situation is similar with other casters. If this was my first character, then I would probably do these quests, but I won’t do this as a fourth alt
